config/ceres_bundle_adjuster.conf

```diff
@@ -66,10 +66,10 @@ ceres:loss_function_type = CAUCHY_LOSS
 ceres:max_num_iterations = 500
 
 # Number of threads to use in the linear solver
-ceres:num_linear_solver_threads = 2
+ceres:num_linear_solver_threads = 8
```

Comment:
@bclipp-kitware Do you recall if this actually made a performance difference?  At one point I tested this and the Ceres solver code could not effectively make use of more than 2 cores anyway.  However, that was on Linux and likely an older version of Ceres.